fastly/blobstorage_test.go

```diff
@@ -15,21 +15,21 @@ func TestClient_BlobStorages(t *testing.T) {
 	var bs *BlobStorage
 	record(t, "blobstorages/create", func(c *Client) {
 		bs, err = c.CreateBlobStorage(&CreateBlobStorageInput{
-			ServiceID:       testServiceID,
-			ServiceVersion:  tv.Number,
-			Name:            "test-blobstorage",
-			Path:            "/logs",
-			AccountName:     "test",
-			Container:       "fastly",
-			SASToken:        "sv=2015-04-05&ss=b&srt=sco&sp=rw&se=2019-07-21T18%3A00%3A00Z&sig=3ABdLOJZosCp0o491T%2BqZGKIhafF1nlM3MzESDDD3Gg%3D",
-			Period:          12,
-			TimestampFormat: "%Y-%m-%dT%H:%M:%S.000",
-			GzipLevel:       9,
```

Comment:
re:

> The intention is that if gzip_level is present it overrides any compression_codec value.

I think we need a test that validates when `GzipLevel` _and_ `CompressionCodec` are provided, that `GzipLevel` will take priority.

Let's ensure we have that test scenario defined for all the updated test files.

That said, I don't think I saw any logic in this change-set for implementing GzipLevel taking priority over CompressionCodec, so could you clarify how that's achieved 🤔

Reply:
| That said, I don't think I saw any logic in this change-set for implementing GzipLevel taking priority over CompressionCodec, so could you clarify how that's achieved :thinking:

This prioritization is handled by the API code so we just exercise it here.

Reply:
Ok, I expanded the testing so each applicable logging endpoint test suite exercises the interaction between these two options.